docs(*) retract new grammatical/stylistic errors (#1247)
Kong/docs.konghq.com: templates/how-to.md

### Introduction

> This is an introduction to this how to guide. It should be limited to a few

Author: ```suggestion
This is the introduction to this how-to guide. It should be limited to a few
```

> succinct sentences that introduces what the user will be learning to do. A

Author: ```suggestion
succinct sentences that introduce what the user will be learning to do. A
```

> How-to guide should provide an end-goal, or something that the user will 

Author: ```suggestion
how-to guide should provide an end-goal or otherwise indicate that the user will 
```
